installers/install.sh

```diff
@@ -6,7 +6,7 @@ BASE_FILE_URL="https://state-tool.s3.amazonaws.com/update/state"
 # Name of the installer executable and archive.
 INSTALLERNAME="state-installer"
 # Channel the installer will target
-CHANNEL='release'
+CHANNEL=""
```

Comment:
This will break https://github.com/ActiveState/cli/blob/23e6e69f9e15104508ecb2fe1ce8e58357bd5e78/activestate.yaml#L250